server/scripts/populate.py

```diff
@@ -0,0 +1,179 @@
+# Assumptions: each club has its own events. Many-to-many not supported
+# yet. See example clubs.yml file for example input. db.ini file must be
+# in same directory
+
+from configparser import ConfigParser
+from datetime import datetime
+from os import path
+from enum import Enum
+from marshmallow import Schema, fields, validate
+import pprint
+import uuid
+import psycopg2
+import yaml
+
+filepath = "clubs.yml"
+pp = pprint.PrettyPrinter(indent=2)
```

Comment:
doesn't look like you're using this anymore, you can remove

```suggestion
```

also can remove the pprint import